exporter/containerimage/export.go

```diff
@@ -166,6 +182,8 @@ func (e *imageExporterInstance) Export(ctx context.Context, src exporter.Source)
 	}
 	defer done(context.TODO())
 
+	ctx = imageutil.WithLayerCompression(ctx, e.layerCompression)
```

Comment:
Don't like hiding this into context very much. Can we make this a parameter please

Reply:
Updated.

I don't like it either. But for now, the export option is like pass-through parameter. Add new parameter will change the whole call chain. :joy: